src/core/server/integration_tests/saved_objects/migrations/group3/actions/actions.test.ts

```diff
@@ -1297,7 +1296,12 @@ describe('migration actions', () => {
       const leftResponse = (await readWithPitTask()) as Either.Left<EsResponseTooLargeError>;
 
       expect(leftResponse.left.type).toBe('es_response_too_large');
-      expect(leftResponse.left.contentLength).toBe(3184);
+      // ES response contains a field that indicates how long it took ES to get the response, e.g.: "took": 7
+      // if ES takes more than 9ms, the payload will be 1 byte bigger.
+      // see https://github.com/elastic/kibana/issues/160994
+      // Thus, the statements below account for response times up to 99ms
+      expect(leftResponse.left.contentLength).toBeGreaterThanOrEqual(3184);
+      expect(leftResponse.left.contentLength).toBeLessThanOrEqual(3185);
```

Comment:
Perhaps we can also assert against the "took" value?

Reply:
The thing is we don't have the `took` field (+ value) when the response is too large, only the error message containing the size of the payload:

https://github.com/elastic/kibana/blob/main/packages/core/saved-objects/core-saved-objects-migration-server-internal/src/actions/read_with_pit.ts#L114